csrank/core/feta_linear.py

```diff
@@ -62,8 +63,8 @@ def _construct_model_(self, n_objects):
 
     def step_decay(self, epoch):
```

Comment:
Same here - this code is basically duplicated and difficult to maintain.

Reply:
Can you suggest a place to keep it, this function is only used by the linear versions of the FETA and FATE. Secondly, we have to change the architecture to tensorflow 2.0. Can we merge these changes so that while converting we have everything to be changed?